app/controllers/api/services_controller.rb

```diff
@@ -67,6 +68,14 @@ def destroy
 
   protected
 
+  def service_discovery_available?
```

Comment:
Can we also use this at https://github.com/3scale/porta/blob/b0940c01de323748f9b98db6a066f41eb9bac086/app/views/api/metrics/index.html.slim#L19?

Reply:
I did not touched this part yet. I guess so. It will be done on the save/refresh of services part of this PR